src/storage.cc

```diff
@@ -49,7 +50,25 @@ void Storage::InitFrom(cyclus::QueryableBackend* b) {
 //- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 void Storage::EnterNotify() {
   cyclus::Facility::EnterNotify();
-  buy_policy.Init(this, &inventory, std::string("inventory"), throughput, active_buying, dormant_buying);
+  inventory_tracker.set_capacity(max_inv_size);
+  if (reorder_point < 0) {
+    buy_policy.Init(this, &inventory, std::string("inventory"),
+                    &inventory_tracker, throughput);
+  }
+  else if (reorder_quantity > 0) {
+    if (reorder_point + reorder_quantity > max_inv_size) {
```

Comment:
What if the inventory drops to 0 in a given time step, isn't it OK for the `reorder_quantity` to completely fill the buffer?  This limit prevents that.  Is this a standard limit of RQ policies?  Or can it be implemented with orders up to and including size Q, but limited by receiving capacity?

Reply:
We treat `reorder_quantity` as a quantized request. So if `inventory` is **not** zero, then the re-order would necessarily put the buffer over the max. So I think if you want the inventory policy to try to fill the buffer, you shouldn't be using (R,Q) inventory anyway. If you don't submit a `reorder_quantity`, then the policy is (s,S) and the order size is fluid and based on the space to reach the max